polygonal_search.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def upsample_array(input_array, num_points=TOTAL_NUM_POINTS):
</pre><p>Maybe rename to "oversample"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/notebooks_lf/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>